templates/engage/cloud-fray.html

```diff
@@ -0,0 +1,34 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}How to face the challenges of becoming a cloud operator{% endblock %}
```

Comment:
for the new engage pages we are supposed to use {% extends_with_args .... implemented by David https://github.com/canonical-web-and-design/ubuntu.com/pull/5255/files